### ISSUE TITLE
extra blank lore

### ISSUE THREAD

**Issue by madtomic** [MLG-Fortress/CustomItemRegistry#19]
How do I get rid of the extra blank lore for recipe for a custom item?
I am using v1.4.5 as I am still on 1.12.2
Any help would be wonderful. Thank you

Edit: I found out it used by the useInvisibleID.
Is there a way not to show this on the item at all while maintain the custom recipe functions?

**Comment by madtomic** [comment]
I found out it used by the useInvisibleID. Is there a way not to show this on the item at all?

**Comment by RoboMWM** [comment]
No, this is how it identifies the item. There is another way, but it requires going outside the Bukkit API...
